### ISSUE TITLE
[FE] Dot Indicator Research: Prep prototype for study

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#74979]
## Background
We need to prep the prototype for the dot indicator/badge study. We'll be using codepen for this. There may already be something that can be used from the Profile study of the federal header so we should check/review that before starting something fresh.

## Tasks
- [x] Review research study needs with Matt (as needed) to make sure we know what's needed in the prototype
- [x] Review existing codepen artifacts from Profile to see if there's anything we can leverage
- [x] Set up codepen for study
- [ ] Review as needed with Matt

## Links
- [Figma mockups](https://www.figma.com/file/15yOY4VEzitxm5tRMDiAzz/My-VA?type=design&node-id=196-26640&mode=design&t=WlUg2EK0DC3jTZkK-0)
- [CodePen collection](https://codepen.io/collection/oEmrJy)

## Acceptance Criteria
- [ ] Prototype is ready for use in the study

## Validation
- [ ] Perform mock walk through of study to ensure it's working properly

**Comment by ajakabcin** [comment]
Work will continue on this next sprint but in a separate ticket since the initial scope of this work is complete
